### ISSUE TITLE
Support for multiple meshes in a single obj file

### ISSUE THREAD

**Issue by jwnimmer-tri** [RobotLocomotion/drake#17266]
In some cases (e.g., some kinds of collision queries), Drake requires that a user decompose their non-convex mesh geometry into convex pieces.  Some third-party tools for doing that create their output as a single `*.obj` file that contains multiple meshes.  When the user tries to load that into Drake, they see this error:

```
RuntimeError: The OBJ file contains multiple objects; only OBJs with a single object are supported.
```

The work-around would be to break into one file per mesh and update the SDFormat (or etc.) file to add multiple geometries, but it would be much more convenient if we could stick with the single OBJ file.

**Comment by jwnimmer-tri** [comment]
One question worth checking: do meshlab, rviz, drake_visualizer support multi-mesh input format?  If they don't, we might need to be careful how we resolve this.

**Comment by SeanCurtis-TRI** [comment]
See #20185 for the bulleted list that is resolving this issue.

**Comment by SeanCurtis-TRI** [comment]
Closed by #21745
